tape/src/test/java/com/squareup/tape/QueueFileTest.java

```diff
@@ -68,6 +68,41 @@
     assertThat(queue.peek()).isEqualTo(expected);
   }
 
+  @Test public void testAddElementTooLarge() throws IOException {
+    QueueFile queue = new QueueFile(file);
+    byte[] newBigValue = new byte[QueueFile.MAX_ELEMENT_SIZE + 1];
```

Comment:
Consider making it possible for tests to change MAX_ELEMENT_SIZE instead of creating 2Gb arrays in tests?

Reply:
e.g. could be constructor param etc.

Reply:
brilliant :)